docker/build-and-push-images.yml

```diff
@@ -326,7 +326,7 @@ stages:
           inputs:
             targetType: "inline"
             script: |
-              Invoke-RestMethod -Uri "$(PROD_PORTAL)/api/RedeployInstance" -Method POST -Body "{ apiKey: '$(PROD_PORTAL_API_KEY)', instanceId: $(STABLE_TEST_INSTANCE_ID) }" -ContentType "application/json"
+              Invoke-RestMethod -Uri "$(DEV_PORTAL)/api/RedeployInstance" -Method POST -Body "{ apiKey: '$(DEV_PORTAL_API_KEY)', instanceId: $(STABLE_TEST_INSTANCE_ID) }" -ContentType "application/json"
```

Comment:
Is this change expected or it was committed by error?

Reply:
Intended

Reply:
Fixed version update on stable